src/cgame/cg_consolecmds.c

```diff
@@ -2317,15 +2317,36 @@ static void CG_ShowEditComponentHelp()
 	CG_Printf("\n\nAvailable ^3<field> ^7:\n\n%s\n", str);
 }
 
+static void CG_ShowEditComponentStyleHelp(const hudComponentFields_t *compField)
```

Comment:
It should mention the current value as well when printing the help. Besides that I noticed something that is not part of this PR, but should be looked at. When not supplying a value e.g. `editcomponent compass style` it shows the help correctly, but it also shows a argument parsing error `int field component needs at least 1 argument <value>`. But when supplying a value out of range or wrong type(string vs int) it does not give a warning or error.